module/min/minwebgpu/src/error.rs

```diff
@@ -22,6 +26,20 @@ mod private
     ConfigurationError( String )
```

Comment:
Can `ConfigurationError` be decomposed on more concrete types?

Reply:
No, context.configure just return jsvalue as a error.